htslib/vcf.h

```diff
@@ -427,18 +449,31 @@ set to one of BCF_ERR* codes and must be checked before calling bcf_write().
 
         Use bcf_write() instead
     */
+    HTSLIB_EXPORT
     int vcf_write(htsFile *fp, const bcf_hdr_t *h, bcf1_t *v) HTS_RESULT_USED;
 
     /** Helper function for the bcf_itr_next() macro; internal use, ignore it */
+    HTSLIB_EXPORT
     int bcf_readrec(BGZF *fp, void *null, void *v, int *tid, hts_pos_t *beg, hts_pos_t *end);
 
+    /// Write a line to a VCF file
+    /** @param line   Line to write
+        @param fp     File to write it to
+        @return 0 on success; -1 on failure
 
+        @note No checks are done on the line being added, apart from
+              ensuring that it ends with a newline.  This function
+              should therefore be used with care.
+    */
+    HTSLIB_EXPORT
+    int vcf_write_line(htsFile *fp, kstring_t *line);
```

Comment:
This function didn't previously have a public API.

Hence this isn't just tying down the API but also expanded it.  I don't mind, but please mention this in the commit message along with a reason (eg who uses it).

It's not used by Samtools or Bcftools.

Reply:
I see _bcftools/vcfconcat.c_ has declared it itself since 2014. Aaargh!